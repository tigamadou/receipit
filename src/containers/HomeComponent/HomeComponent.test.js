import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import storeReducer from '../../redux/store';
import Home from './HomeComponent';

const store = storeReducer();
describe('Home component tests', () => {
  test('renders home component ', () => {
    render(
      <Provider store={store}>
        <Home />
        {' '}
      </Provider>,
    );
    expect(screen.getByTestId('home')).toBeInTheDocument();
  });

  test('should match with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Home />
          {' '}
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
