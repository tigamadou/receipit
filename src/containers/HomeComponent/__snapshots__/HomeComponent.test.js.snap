// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component tests should match with snapshot 1`] = `
Array [
  <div
    className="container"
    data-testid="home"
  >
    <div
      className="cards"
    />
  </div>,
  " ",
]
`;
